docs-ref-services/logic-apps.md

```diff
@@ -28,17 +28,27 @@ The advantages of using Logic Apps include the following:
 
 Logic Apps is a fully managed iPaaS (integration Platform as a Service) allowing developers not to have to worry about building hosting, scalability, availability and management. Logic Apps will scale up automatically to meet demand.
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure Logic Apps modules for Node.js
-
-### Management
+#### Install npm modules
 ```bash
 npm install azure-arm-logic
 ```
 
-##Example
+### Example
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const LogicManagement = require('azure-arm-logic');
+
+msRestAzure
+  .interactiveLogin()
+  .then(credentials => {
+    const client = new LogicManagement(credentials, 'subscription-id');

```

Comment:
define a `subscriptionId` to be consistent moving forward. i think separate variables are clearer